### PR TITLE
Bottou2010large+elampietti

### PR DESCRIPTION
**Pull request by elampietti**

### CODE REVIEW

File: _summaries/bottou2010large_1.md [hunk 1]
```diff
@@ -1,19 +1,47 @@
...
+The following diagram reveals how the stochastic gradient descent algorithm drastically outperforms the other algorithms in very little time and is only overtaken by TRON after the expected risk stops improving.
+
+![results 1](https://user-images.githubusercontent.com/7085644/132602485-05395855-1570-4c02-b3c3-bce60b15ab8d.PNG)
+
+The next two diagrams reveal how the other SGD variants (ASGD & SGDQN) perform even better compared to  CRF L-BFGS which took 72 minutes instead of a couple minutes to achieve the same results.
+
+![results 2](https://user-images.githubusercontent.com/7085644/132602491-29a29d44-b014-471a-aaa2-86f409bff077.PNG)
+![results 3](https://user-images.githubusercontent.com/7085644/132602497-8158b999-5ce0-470a-af9c-df4862a1f51d.PNG)
 
 ## TL;DR
```
Comment (patelojas) [inline _summaries/bottou2010large_1.md:44]: I think it might be valuable to add a little more on comparison on SGD vs GD.
